src/open_company/resources/common.clj

```diff
@@ -73,6 +74,21 @@
   [resource]
   (apply dissoc resource reserved-properties))
 
+(defn- name-for
+  "Replace :name in the map with :real-name if it's not blank."
+  [user]
+  (if (s/blank? (:real-name user))
+    user
+    (assoc user :name (:real-name user))))
+
+(defn author-for-user
+  "Extract the :avatar/:image, :user-id and :name (the author fields) from the JWToken claims."
+  [user]
+  (-> user
+    (name-for)
+    (select-keys [:avatar :user-id :name])
```

Comment:
@belucid i think the problem with the author images is here, changing this line with this
```
(select-keys [:avatar :user-id :name :image])
```
and reimporting all the companies worked for me. I leave the change to you to be sure it's the right fix.